cime_config/acme/allactive/testlist_allactive.xml

```diff
@@ -1,289 +1,5 @@
 <?xml version="1.0"?>
 <testlist>
-  <compset name="A">
```

Comment:
Are you deleting this stuff because ACME uses update_acme_tests to manage tests?